prow/config/plugins.yaml

```diff
@@ -44,6 +44,17 @@ plugins:
     plugins:
     - config-updater
 
+approve:
+  - repos:
+    - metal3-io
+
+    # RequireSelfApproval requires PR authors to explicitly approve their PRs.
+    # Otherwise the plugin assumes the author of the PR approves the changes in the PR.
+    require_self_approval: true
+
+    # A /lgtm from a single approver should not allow a PR to merge.
+    lgtm_acts_as_approve: false
```

Comment:
Wasn't this already the case by default ? but +1 to explicitly disable it.

Reply:
It's possible the default changed when you upgraded the prow cluster, but as of a few weeks ago it was definitely enabled. See e.g. https://github.com/metal3-io/baremetal-operator/pull/852 where the bot adds the approved label as soon as the PR is opened.

Reply:
I see, then I misunderstood `lgtm_acts_as_approve` a bit, because I though adding lgtm would even approve the PR. It is true that if you are an approver in `OWNERS` file, then you will get approved label immediately when you open a PR and it was the same from the beginning of the usage of Prow.

Reply:
There are 2 aspects of this and they're closely related. I may not have clearly explained things in the comments, so let me try to clarify here and then we can update the file with a better description.

The default for `require_self_approval` is false, which means then someone in the approver list submits a PR that PR is automatically approved. The effect of that is that they then only need 1 person to leave a `/lgtm` comment for the PR to merge. We want it set to true so all PRs require an explicit `/approve` comment as well.

The default for `lgtm_acts_as_approve` is false, which means that an approver who comments `/lgtm` does not also implicitly add the approved label. We do not want the value set to true, because that would also introduce a path for a PR to be merged by a single person. It's redundant, but I'm setting it explicitly to false here so the change in the file will be associated with the git commit message linking to the discussion of why we want it that way.

Reply:
Got it, thanks for explanation @dhellmann

Reply:
Sorry, I got confused about which line this comment was referring to :)

I thought that we had `lgtm_acts_as_approve` on as well (which is why I mostly avoid doing /lgtm), but perhaps that was on our downstream repo.